### PR TITLE
Update dependencies and fix compatibility with new Flask

### PR DESCRIPTION
**Pull request by frenzymadness**
Before I try to find a new home for this application.

### CODE REVIEW

**Comment by frenzymadness** [comment]
It seems that I also have to switch it to Github Actions.

**Comment by hroncok** [comment]
Feel free to drop the PyPI upload support from the CI.

**Comment by frenzymadness** [comment]
All green. There is one change we might need to discuss – the path to string change in URL for username. We can do that tomorrow.

File: tests/test_status.py [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
 
-@pytest.mark.parametrize('kojiurl', ('https://koji.fedoraproject.org/',
-                                     'https://cbs.centos.org/',
-                                     'http://koji.rpmfusion.org/'))
-def test_different_kojis_work(betamax_parametrized_session, kojiurl):
-    username = 'churchyard' if 'fedoraproject' in kojiurl else 'kojira'
+@pytest.mark.parametrize(
+    ('kojiurl', 'username'),
+    (
+        ('https://koji.fedoraproject.org/', 'churchyard'),
```
Comment (hroncok) [inline tests/test_status.py:9]: Do we want to add something with a slash? e.g. [distrobuildsync-eln/jenkins-continuous-infra.apps.ci.centos.org](https://koji.fedoraproject.org/koji/userinfo?userID=5159)

File: tests/test_webapp.py [hunk 2]
```diff
@@ -19,10 +19,18 @@ def test_webapp_returns_200_for_good_username(response):
...
 
 
-def test_webapp_redirects_without_trailing_slash(testapp):
+def test_webapp_works_with_and_without_trailing_slash(testapp):
     response = testapp.get('/churchyard')
-    assert response.status_code // 100 == 3
-    assert response.headers['Location'].endswith('/churchyard/')
+    assert response.status_code == 200
+    response_with_trailing_slash = testapp.get('/churchyard/')
+    assert response_with_trailing_slash.status_code == 200
```
Comment (hroncok) [inline tests/test_webapp.py:26]: Do we assert the content is equal?

File: tests/test_status.py (hunk 1 above)
Comment (frenzymadness) [reply tests/test_status.py:9]: If the implementations of the Koji are the same, we have it covered in `test_webapp_works_with_slash_in_username`.

File: tests/test_webapp.py (hunk 2 above)
Comment (frenzymadness) [reply tests/test_webapp.py:26]: I plan to do that but was afraid about a race conditions when a new build appears in the meantime. But the responses are recorded as cassettes so is it useful?

Comment (hroncok) [reply tests/test_webapp.py:26]: Well, if the Koji responses are recorded, the race condition can only happen when re-recording them, no?
The assert would make sure that both URLs go and fetch the same data from Koji.

Comment (frenzymadness) [reply tests/test_webapp.py:26]: Added.

**Review by hroncok — APPROVED** [review]
Thanks.

**Pull request merged**